deploy/028_deploy_linea_adapter.ts

```diff
@@ -14,7 +14,9 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       WETH[chainId],
       L1_ADDRESS_MAP[chainId].lineaMessageService,
       L1_ADDRESS_MAP[chainId].lineaTokenBridge,
-      L1_ADDRESS_MAP[chainId].lineaUsdcBridge,
+      // TODO: Requires USDC Linea address to be added, currently only USDC.e is set for Linea
```

Comment:
This should be the same address because Linea USDC is Circle's upgradable USDC standard.